### PR TITLE
test: add vhs test page for netlify testing

### PR DESCRIPTION
**Pull request by brandonocasey**
This allows us to test vhs with mux.js changes using netlify

### CODE REVIEW

File: scripts/netlify.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const shell = require('shelljs');
+
+shell.exec('git clone https://github.com/videojs/http-streaming');
```
Comment (brandonocasey) [inline scripts/netlify.js]: didn't see a better way to do this since vhs needs to build mux.js into itself.

Comment (misteroneill) [reply scripts/netlify.js]: Git submodules! :trollface:

File: scripts/netlify.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+shell.cd('http-streaming');
+shell.exec('npm ci');
+shell.exec('npm link ../');
+shell.exec('npm run netlify');
```
Comment (gkatsev) [inline scripts/netlify.js]: can we add a comment to what this is doing? Specifically, that's it's running the netlify build of VHS itself.

File: scripts/netlify.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+shell.cp('-R', 'deploy',  '../');
```
Comment (gkatsev) [inline scripts/netlify.js]: if we wanted to, we could configure it to deploy `http-streaming/deploy` then we don't need these 3 lines.

File: package.json [hunk 4]
```diff
@@ -59,6 +60,7 @@
     "dist/"
   ],
   "devDependencies": {
+    "@videojs/http-streaming": "^1.12.0",
```
[the remark this answers is not in the document]
Comment (brandonocasey) [reply package.json]: nope

File: scripts/netlify.js (hunk 3 above)
Comment (brandonocasey) [reply scripts/netlify.js]: We could, but I think it's better to run these lines and keep netlify `deploy` in a predictable place for all of our projects.

**Pull request merged**